Conclusion & Abstract? (#215)
chhoumann/thesis-chemcam: report_thesis/src/index.tex

\newpage

\begin{abstract}
This thesis advances the analysis of \gls{libs} data for predicting major oxide compositions in geological samples.

> By integrating machine learning techniques and ensemble regression models, the study addresses challenges like high dimensionality, non-linearity, multicollinearity, and limited data availability.

Reviewer: ```suggestion
By integrating machine learning techniques and ensemble regression models, the study addresses challenges like high dimensionality, multicollinearity, and limited data availability.
```

Non linearity is definitely an issue, but we havent highlighted it as a challenge.

> Key innovations include the use of stacked generalization for improved model performance and automated hyperparameter optimization with Optuna.

Reviewer: ```suggestion
Key innovations include the use of stacked generalization for improved model performance and an automated hyperparameter optimization framework.
```
I dont feel like namedropping Optuna here serves us any good. Implementation details.
